src/mono/wasm/runtime/loader/blazor/_Integration.ts

```diff
@@ -71,30 +75,43 @@ export function setupModuleForBlazor(module: DotnetModuleInternal) {
     };
 
     loaderHelpers.downloadResource = downloadResource; // polyfills were already assigned
-    module.disableDotnet6Compatibility = false;
+}
+
+function appendUniqueQuery(attemptUrl: string): string {
```

Comment:
Could we do better ? https://github.com/dotnet/runtime/issues/87280

Reply:
I'm tracking this API. In follow up where we refactor the blazor, I would like to come with something